packages/grid/_modules_/grid/components/panel/GridPanelWrapper.tsx

```diff
@@ -17,13 +17,15 @@ const useStyles = makeStyles(
   { name: 'MuiDataGridPanelWrapper' },
 );
 
+const isEnabled = () => true;
```

Comment:
this one can be moved in the upper scope. So it's not recreated at every render

Reply:
How is it not already the upper scope?

Reply:
Ah yes I thought I saw it in the component.